llvm/include/llvm/ADT/SmallSet.h

```diff
@@ -206,11 +196,12 @@ class SmallSet {
   bool erase(const T &V) {
     if (!isSmall())
       return Set.erase(V);
-    for (mutable_iterator I = Vector.begin(), E = Vector.end(); I != E; ++I)
-      if (*I == V) {
-        Vector.erase(I);
-        return true;
-      }
+
+    auto It = llvm::find(Vector, V);
+    if (It != Vector.end()) {
```

Comment:
nit: this is not used outside of the `if`
```suggestion
    if (auto It = llvm::find(Vector, V); It != Vector.end()) {
```